models/sdn-fabric-0.1.x/sdn-fabric.tree

```diff
@@ -0,0 +1,60 @@
+module: onf-switch
+  +--rw switch* [switch-id]
+     +--rw switch-id             inet:domain-name
+     +--rw display-name?         string
+     +--rw description?          string
+     +--rw attribute* [attribute-key]
+     |  +--rw attribute-key    yt:yang-identifier
+     |  +--rw value            string
+     +--rw model                 -> /sm:switch-model/switch-model-id
+     +--rw role                  enumeration
+     +--rw management
+     |  +--rw address?       inet:ipv4-address-no-zone
+     |  +--rw port-number?   inet:port-number
+     +--rw port* [cage-number channel-number]
+     |  +--rw cage-number           -> /sm:switch-model/port/cage-number
+     |  +--rw channel-number        -> /sm:switch-model/port/channel-number
+     |  +--rw display-name?         string
+     |  +--rw description?          string
+     |  +--rw speed?                identityref
+     |  +--rw dhcp-connect-point?   -> ../../dhcp-connect-point/dhcp-id
+     |  +--rw vlans
+     |     +--rw untagged?   -> ../../../vlan/vlan-id
+     |     +--rw tagged*     -> ../../../vlan/vlan-id
+     +--rw switch-pair
+     |  +--rw cage-number?      -> ../../port/cage-number
+     |  +--rw channel-number?   -> ../../port/channel-number
+     |  +--rw paired-switch?    -> /switch/switch-id
+     +--rw vlan* [vlan-id]
+     |  +--rw vlan-id         st:vlan
+     |  +--rw display-name?   string
+     |  +--rw description?    string
+     |  +--rw subnet          inet:ipv4-prefix
+     +--rw dhcp-connect-point* [dhcp-id]
+        +--rw dhcp-id          yt:yang-identifier
+        +--rw connect-point*   inet:ipv4-address-no-zone
+
+module: onf-switch-model
+  +--rw switch-model* [switch-model-id]
+     +--rw switch-model-id    yt:yang-identifier
+     +--rw display-name?      string
+     +--rw description?       string
+     +--rw attribute* [attribute-key]
+     |  +--rw attribute-key    yt:yang-identifier
+     |  +--rw value            string
+     +--rw form-factor        enumeration
+     +--rw pipeline?          enumeration
+     +--rw port* [cage-number channel-number]
+        +--rw cage-number       uint8
+        +--rw channel-number    uint8
+        +--rw display-name?     string
+        +--rw description?      string
+        +--rw speeds*           identityref
+
+module: onf-route
+  +--rw route* [route-id]
```

Comment:
are we adding a display-name and description for routes?

Reply:
added